kubernetes/linux/setup.sh

```diff
@@ -30,27 +40,27 @@ sudo apt-get install jq=1.5+dfsg-2 -y
 #used to setcaps for ruby process to read /proc/env
 sudo apt-get install libcap2-bin -y
 
-wget https://dl.influxdata.com/telegraf/releases/telegraf-1.18.0_linux_amd64.tar.gz
-tar -zxvf telegraf-1.18.0_linux_amd64.tar.gz
+wget https://dl.influxdata.com/telegraf/releases/telegraf-1.18.0_linux_$ARCH.tar.gz
```

Comment:
There are bunch of new changes  in ci_dev, please merge your changes with ci_dev branch changes.

Reply:
done